person_capture/gui_app.py

```
@@ -3219,10 +3408,23 @@ def _run_deferred_probe_after_prescan(cap_obj, fps_val, total_val, hdr_is_active
+                if str(getattr(cfg, "prescan_cache_mode", "auto") or "auto").lower() != "refresh":
+                    prescan_loaded, keep_spans, cached_ref_face_feat, _cache_meta = self._load_prescan_cache(
+                        cfg, float(fps), int(total_frames)
+                    )
+                    if prescan_loaded:
+                        ref_face_feat = cached_ref_face_feat
+                if not prescan_loaded:
+                    mode = str(getattr(cfg, "prescan_cache_mode", "auto") or "auto").lower()
+                    suffix = " (refresh cache)" if mode == "refresh" else ""
+                    self._status(f"Pre-scan.{suffix}", key="phase", interval=2.0)
+                    keep_spans, ref_face_feat = self._prescan(
+                        cap, int(round(fps)), total_frames, face, ref_face_feat, cfg
```

Ruff (0.15.11)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)